app/browsered/cookie-banner.js

```diff
@@ -0,0 +1,265 @@
+'use strict'
+
+const analytics = require('./analytics')
```

Comment:
Can we split the cookie functions out?
And require it in?

Reply:
Added new file for cookie functions